docs/source/mistral.rst

```diff
@@ -59,64 +55,33 @@ Let's say we need to upgrade and reboot all the members of a MongoDB replica set
                 - duration
```

Comment:
It would be nice to make this worklfow actually working against 'mock' actions (make them part of example),  run this workflow as part of the test, and `literanlinclude` it in the docs.